services/scabbard/libscabbard/src/store/scabbard_store/diesel/models.rs

```diff
@@ -283,6 +279,51 @@ impl
             "WAITINGFORVOTE" => Scabbard2pcState::WaitingForVote,
             "ABORT" => Scabbard2pcState::Abort,
             "COMMIT" => Scabbard2pcState::Commit,
+            "WAITINGFORVOTEREQUEST" => Scabbard2pcState::WaitingForVoteRequest,
+            "VOTED" => {
+                let vote = context
+                    .vote
+                    .as_ref()
+                    .map(|v| match v.as_str() {
+                        "TRUE" => Some(true),
+                        "FALSE" => Some(false),
+                        _ => None,
+                    })
+                    .ok_or_else(|| {
+                        InternalError::with_message(
+                        "Failed to convert to ScabbardContext, context has state 'voted' but vote
+                        is unset"
+                        .to_string(),
+                    )
+                    })?
+                    .ok_or_else(|| {
+                        InternalError::with_message(
+                            "Failed to convert to ScabbardContext, context has state 'voted' but an
```

Comment:
if the lines need to be wrapped they should have a backslash at the end of the first line, otherwise an unintended new line is added to the string (same for several other error messages)

Reply:
looks like this has been fixed in other parts of this commit but not in the two "Failed to convert to ScabbardContext" errors in this file

Reply:
Thought I got all of those, should be fixed now